src/widgets/modals/DispensaryLookup.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
```

Comment:
:nit 2020 😁

Reply:
Oops!  Would fix, but this component is removed in follow-up PRs :sweat_smile:!